pkgs/ha-dyson/default.nix

```diff
@@ -20,11 +23,6 @@ with home-assistant.python.pkgs; buildHomeAssistantCustomComponent rec {
     setuptools
   ];
 
-  installPhase = ''
-    mkdir -p $out/custom_components
-    cp -r custom_components/dyson_local $out/custom_components/
-  '';
-
```

Comment:
@mweinelt this is an example of one with `format=pyproject` but I still want to just copy the component out of the source (the pyproject-based install doesn't install the manifest).

Reply:
I think we can try submitting fixes upstream in such cases.

But note that the packaging aspect on this package is pretty minimal. They're only configuring tools from `setup.cfg` and `pyproject.toml` right now, and our PEP517 builder automatically falls back to setuptools with some defaults.

```patch
diff --git a/setup.cfg b/setup.cfg
index a4a6350..c1f3f25 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,3 +1,7 @@
+[options.package_data]
+* =
+  manifest.json
+
 [flake8]
 exclude = .venv,.git,.tox,docs,venv,bin,lib,deps,build
 doctests = True
```